docs/Dependencies.md

```diff
@@ -1,3 +1,5 @@
+# TODO: REWRITE IF CHANGES DEEMED ACCEPTABLE
```

Comment:
@KingDarBoja do you have any thoughts on the changes here?

Reply:
I have nothing against it, as long as there aren't API changes but only moving those dependency definitions into each owning directories. Based on that, pretty much point out that now eachdependency lives on the same file as the function fiels. ⭐